tests/reducers/AppReducer.spec.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @group reducers
+ */
+
+import { ACTIONS } from "../../src/enums/AppDispatchActions";
+import { AppReducer } from "../../src/reducers/AppReducer";
+import { IBlock, IMainUser, ITransaction, IUser } from "../../src/typings/AppTypes";
+
+const { initialState } = global;
+
+const trans: ITransaction = {
+  to: "B",
+  from: "C",
+  amount: 4.56,
+  message: "Sixth Transaction",
+  signature: "BC4.56"
+};
+
+const user: IMainUser = { publicKey: "newPK", privateKey: "newSK", balance: 101.01 };
+const users: IUser[] = [{ balance: 100.11, publicKey: "LibraCoinKey" }];
+
+const block: IBlock = {
+  index: 1,
+  prevHash: new Array(64).fill("0").join(""),
+  currHash: new Array(64).fill("1").join(""),
+  transactions: [trans],
+  timestamp: Date.parse("05/15/2021"),
```

Comment:
Improper format here, should be similar: "01 May 2021 00:00:00 UTC".

This could lead to differences between GitHub Actions and local.